components/expenses/ExpensesList.js

```diff
@@ -27,6 +40,13 @@ const ExpensesList = ({
 }) => {
   expenses = !isLoading ? expenses : [...new Array(nbPlaceholders)];
 
+  let totalAmount = 0;
+  if (!isLoading) {
+    expenses.map(expense => {
+      totalAmount += expense.amount;
+    });
+  }
```

Comment:
:red_circle: This should be calculated after the next conditional that checks if expenses exist (50~52).

:ice_cream: Not necessary, but if you're into lodash, this could be simplified with `sumBy`.
```suggestion
  import { sumBy } from 'lodash';
  
  const totalAmount = sumBy(expenses, 'amount');
```

Reply:
Nice suggestion! I tested this approach locally and it worked. I'm pushing the commit now!